app/policies/page_policy.rb

```diff
@@ -36,7 +36,7 @@ def edit?
 
   private
   def user_is_member?
-    (user.is_member? || user.admin?)
+    (user.member? || user.admin?)
   end
 
```

Comment:
Maybe you could refactor this private method away. It is confusing: the method is name `user_is_member?`, but the implementation is actually the same as the already existing `user.is_member_or_admin?`

Reply:
Good point - will do.